index.js

```diff
@@ -10,7 +10,7 @@ async function main() {
     console.log(chalk.blue('🚀 Welcome to Todo CLI!'));
     console.log(chalk.blue('='.repeat(50)));
```

Comment:
Consider using 'while (true)' instead of 'while (true === true)' for simplicity and readability.